mods/ra/maps/evacuation/map.yaml

```diff
@@ -0,0 +1,1757 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: Evacuation
+
+Author: Scott_NZ
+
+Tileset: SNOW
+
+MapSize: 128,128
+
+Bounds: 16,16,96,96
+
+Visibility: MissionSelector, Lobby
```

Comment:
But it doesn't make sense to duplicate the whole map just with different Visibility entries, either. And the map *is* both single- and multi-player capable. I was very careful to allow for that especially.

Either way, the fix is easy: http://stackoverflow.com/a/8317808